Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -372,8 +408,8 @@ private void Install(UserPlugin plugin, string downloadedFilePath)
                 MessageBox.Show(Context.API.GetTranslation("plugin_pluginsmanager_install_errormetadatafile"));
                 return;
             }
-
-            string newPluginPath = Path.Combine(DataLocation.PluginsDirectory, $"{plugin.Name}-{plugin.Version}");
+            var directory = String.IsNullOrEmpty(plugin.Version) ? $"{plugin.Name}" : $"{plugin.Name}-{plugin.Version}";
```

Comment:
this may create some issue. I don't think flow can handle the plugin with same name. @jjw24 any potential solution for this?

Reply:
to solve the problem of downloading plugin of the same name you can append the zip name with a random guid, but will need to test how flow loads it up and how to uninstall duplicate plugins

Reply:
Could I just run the `Uninstall` method with the filename before the `Install` method?

Reply:
You still can run uninstall after install because, in the end it will require a restart. the uninstall places a needdelete.txt file on the plugin to uninstall, so when restarted the to be deleted plugin will be deleted, and your new downloaded plugin will be loaded.

only thing is if you going to do this, might need to prompt the user to let them know, because they might want to try the newer plugin but still wants to keep their existing.

Reply:
So in it's current state if one tries to re-download the same zip flow shows there was an error installing. You can uninstall normally.